internal/support/cronscheduler/infrastructure/interfaces/SchedulerManager.go

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/models"
+)
+
+type SchedulerManager interface {
+	AddScheduleJob(job models.ScheduleJob) errors.EdgeX
+	UpdateScheduleJob(job models.ScheduleJob) errors.EdgeX
+	DeleteScheduleJobByName(name string) errors.EdgeX
+	StartScheduleJobByName(name string) errors.EdgeX
+	StopScheduleJobByName(name string) errors.EdgeX
+	TriggerScheduleJobByName(name string) errors.EdgeX
```

Comment:
```suggestion
	TriggerScheduleJobByName(name, correlationId string) errors.EdgeX
```

Add correlationId to each function and print correlationId to the log

Reply:
New issue opened: https://github.com/edgexfoundry/edgex-go/issues/4852